qt/scientific_interfaces/Inelastic/Processor/ElwinView.cpp

```diff
@@ -205,6 +205,14 @@ void ElwinView::addTableEntry(int row, std::string const &name, std::string cons
   setCell(std::move(cell), row, 1);
 }
 
+void ElwinView::updatePreviewWorkspaceNames(const std::vector<std::string> &names) {
+  disconnectSignals();
+  m_uiForm.cbPreviewFile->clear();
+  m_uiForm.cbPreviewFile->addItems(stdVectorToQStringList(names));
+  m_uiForm.cbPreviewFile->setCurrentIndex(0);
```

Comment:
If you set this line to something like

```suggestion
  m_uiForm.cbPreviewFile->setCurrentIndex(names.size() - 1);
```
Then that should trigger the correct signal